src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/preview/main.js

```diff
@@ -246,14 +72,17 @@ define(['app-config'], function(AppConfig) {
                 if (!!this.initiated) {
                     var changes = {};
 
-                    if (this.method === 'ws') {
-                        ws.update.call(this, changes);
-                    } else {
-                        ajax.update.call(this, changes);
-                    }
+                    return this.client.send(MESSAGE_HANDLER_NAME, {
+                        command: 'update',
+                        data: changes
+                    });
                 }
             },
 
+            updateIframe = function(handler, message) {
```

Comment:
It does not necessarily have to be updating the iFrame, right? I guess it could also be an own window.